pkg/operation/botanist/logging.go

```diff
@@ -162,8 +162,8 @@ func (b *Botanist) destroyShootNodeLogging(ctx context.Context) error {
 }
 
 func (b *Botanist) isShootNodeLoggingEnabled() bool {
-	if b.Shoot != nil && b.Shoot.IsLoggingEnabled() && b.Config != nil &&
-		b.Config.Logging != nil && b.Config.Logging.ShootNodeLogging != nil {
```

Comment:
The `!= nil` checks seem still reasonable

Reply:
Adding `!=nil` is not required but It will make things clear. I will add it. Thanks

Reply:
Addressed in 8c9ee3ad536361a54bdf56825cd1cb78a2bc8b72